src/chaosinventory/base/views/api.py

```diff
@@ -0,0 +1,62 @@
+from rest_framework import viewsets
+
+from ..models import (
+    DataType, Entity, InventoryIdSchema, Item, ItemInventoryId, Location,
+    LocationData, Product, ProductInventoryId, Tag,
+)
+from ..serializers import (
+    DataTypeSerializer, EntitySerializer, InventoryIdSchemaSerializer,
+    ItemInventoryIdSerializer, ItemSerializer, LocationDataSerializer,
+    LocationSerializer, ProductInventoryIdSerializer, ProductSerializer,
+    TagSerializer,
+)
+
+
+class InventoryIdSchemaViewSet(viewsets.ModelViewSet):
```

Comment:
When deleting an `InventoryIdSchema` (and also others) which is used in another (or the same) model a 500 error is returned. It would be nice to have a readable error message. This could be also resolved after merging this great PR.